lib/phi_attrs/phi_record.rb

```diff
@@ -225,6 +226,36 @@ def __user_id_string(access_list)
         access_list ||= []
         access_list.map { |c| "'#{c[:user_id]}'" }.join(',')
       end
+
+      def i18n_reason
+        controller = RequestStore.store[:phi_attrs_controller]
+        action = RequestStore.store[:phi_attrs_action]
+
+        return nil if controller.blank? || action.blank?
+
+        i18n_path = ['phi'] + __path_to_controller_and_action(controller, action)
```

Comment:
Instead of a hard coded `'phi` we could configure this like the `log_path` so that application users can customize.

Reply:
Hm. Sure...I've never seen it done that way in a rails app. It shouldn't be hard.

Reply:
I would say we don't need to worry about this yet. I think there's also some benefit to having this in a defined place.

Reply:
Let's make that an issue: #36